File: Java/BitwiseOperator/XOR1.java
////////////////////////////////////////////////////////////////////////////////////////////////////
//
//  Accept a decimal number from user, apply bitwise XOR operation with iMask,
//  and display the updated number.
//
//  iMask = 0X00000084
//  
//
////////////////////////////////////////////////////////////////////////////////////////////////////


import java.util.*;

public class XOR1 
{
  public static void main(String A[])
  {
    Scanner sobj = new Scanner(System.in);

    int iNo = 0;
    int iMask = 0X00000084;
    int iAns = 0;

    System.out.println("Enter Number : ");
    iNo = sobj.nextInt();

    iAns = iNo ^ iMask;

    System.out.println("Updated Number is : "+iAns);
  }
}

File: Python/BitwiseOperator/XOR1.py
######################################################################################################################
##
##   Accept a decimal number from user, apply bitwise XOR operation with iMask, and display the updated number.
##
##   iMask = 0X00000084
##
######################################################################################################################

iNo = 0;
iMask = 0X00000084;
iAns = 0;

iNo = int(input("Enter Number : "));

iAns = iNo ^ iMask;

print("Updated Number is : ",iAns);